unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -39,14 +40,24 @@ export class KeyList extends React.Component {
           <Pagination> takes a list of items
           and a function that takes list of items and renders the list.
           This is so that the Pagination component can be reused across the app
-      */}
-        <Pagination
-          items={keys}
-          currentPage={page + 1}
-          itemCount={lock.outstandingKeys}
-          renderItems={renderItems}
-          goToPage={loadPage}
-        />
+          */
+        keys.length > 0 ? (
+          <Pagination
+            items={keys}
+            currentPage={page + 1}
+            itemCount={lock.outstandingKeys}
+            renderItems={renderItems}
+            goToPage={loadPage}
+          />
+        ) : (
+          <Message>
+            No keys have been purchased yet.{' '}
```

Comment:
I have used `&nbsp;` instead of `{' '}` but I do not have strong feelings about it.
Feel free to change or leave as is.